packages/react-storage/src/components/StorageBrowser/Views/Controls/Navigate.tsx

```diff
@@ -17,9 +17,7 @@ import { CLASS_BASE } from '../constants';
 
 import { LocationData, useAction } from '../../context/actions';
 
-interface NavigateItemProps extends ButtonElementProps {
-  seperator?: React.ReactNode;
```

Comment:
I removed this prop since with this change we can always render the `<Seperator />`